spec/features/users/01_user_create_account_spec.rb

```diff
@@ -11,52 +11,55 @@
   # [ ] Form returns error when required fields are not entered correctly
   # [ ] Displays success message and redirects to user profile when form
   #     is submitted correctly
+  context "as a new user" do
 
-  scenario 'user sees sign-up form' do
-    visit root_path
-    click_link("Sign Up")
+    before(:each) do
+      visit root_path
+      click_link("Sign Up")
+    end
 
-    expect(page).to have_css("form#new_user")
-  end
+    scenario "I can visit the root path and click a link to create a new
+      account" do
 
-  scenario 'user sees all inputs form' do
-    visit root_path
-    click_link("Sign Up")
+      expect(page).to have_css("form#new_user")
+    end
 
-    expect(page).to have_css("input#user_first_name")
-    expect(page).to have_css("input#user_last_name")
-    expect(page).to have_css("input#user_email")
-    expect(page).to have_css("input#user_password")
-    expect(page).to have_css("input#user_password_confirmation")
-  end
+    scenario "I can click a link to create a new account and is taken to the
+      new user form" do
 
-  scenario "user fills in form correctly" do
-    visit root_path
-    click_link("Sign Up")
+      expect(page).to have_css("input#user_first_name")
+      expect(page).to have_css("input#user_last_name")
+      expect(page).to have_css("input#user_email")
+      expect(page).to have_css("input#user_password")
+      expect(page).to have_css("input#user_password_confirmation")
+    end
 
-    fill_in("First Name", with: "John")
-    fill_in("Last Name", with: "Smith")
-    fill_in("Email", with: "abc@gmail.com")
-    fill_in("Password", with: "password")
-    fill_in("Password Confirmation", with: "password")
-    click_button('Sign Up')
+    scenario "I will be taken to the new user form and successfully create an
+      account by filling out the form correctly" do
 
-    expect(page).to have_content("You have successfully signed up!")
-  end
+      fill_in("First Name", with: "John")
+      fill_in("Last Name", with: "Smith")
+      fill_in("Email", with: "abc@gmail.com")
+      fill_in("Password", with: "password")
+      fill_in("Password Confirmation", with: "password")
+      click_button('Sign Up')
+
+      expect(page).to have_content("You have successfully signed up!")
+    end
 
-  scenario "user fills in form correctly" do
-    visit root_path
-    click_link("Sign Up")
+    scenario "I will not be able to create a new account if I fill out the
```

Comment:
Not only will you not be able to create an account, but you will also see an alert. A side effect of this test is that it is checking form validations/ your alerts. Again, try to shorten this by either rephrasing or nesting statements